lib/midi_synth/command.ex

```diff
@@ -28,27 +28,30 @@ defmodule MIDISynth.Command do
   @typedoc "A MIDI program"
   @type program :: 1..128
 
+  @typedoc "The MIDI channel number"
+  @type channel :: 0..15
+
   @doc """
   Press a key down
   """
-  @spec note_on(note(), velocity()) :: binary()
-  def note_on(note, velocity) do
-    <<0x90, note, velocity>>
+  @spec note_on(note(), velocity(), channel()) :: binary()
+  def note_on(note, velocity, channel) do
+    <<0x90 + channel, note, velocity>>
```

Comment:
```suggestion
    <<0x9::4, channel::4, note, velocity>>
```

This is personal preference to use binary notation to spell out the bits. If nothing else, if channel is out of range, it won't change the command.

Reply:
Making `channel` the first parameter feels better to me even though it breaks the API. If it feels better to you, I say we change it. If not, I'm fine with it as it stands too.